source/main/physics/Beam.cpp

```diff
@@ -6122,9 +6122,9 @@ bool Beam::LoadTruck(
 	}
 
 	// Calculate the approximate median
-	std::vector<Real> mx(free_node, 0.0f);
-	std::vector<Real> my(free_node, 0.0f);
-	std::vector<Real> mz(free_node, 0.0f);
+	std::vector<Ogre::Real> mx(free_node, 0.0f);
+	std::vector<Ogre::Real> my(free_node, 0.0f);
+	std::vector<Ogre::Real> mz(free_node, 0.0f);

```

Comment:
Real can be found here: https://github.com/RigsOfRods/rigs-of-rods/blob/master/source/main/physics/collision/CartesianToTriangleTransform.h#L32
And here: https://bitbucket.org/sinbad/ogre/src/7343503880d45d110ef2ab98c3a5e82b76b29eb1/OgreMain/include/OgrePrerequisites.h?at=v1-9&fileviewer=file-view-default#OgrePrerequisites.h-70